api/src/main/java/org/openmrs/module/metadatadeploy/bundle/CoreConstructors.java

```diff
@@ -56,6 +60,16 @@
 public class CoreConstructors {
 
 	/**
+	 * Constructs a dbsessionFactory source
+	 * @return the transient object
+	 */
+	public static DbSessionFactory dbSessionFactory(SessionFactory sessionFactory) {
```

Comment:
Can you point me to the section, on that page, which instructs you to include DBSessionFactory in CoreConstructor?

Did you look and understand the sample commit i gave you?